types/web-element.d.ts

```diff
@@ -177,6 +177,8 @@ export interface ScopedElement extends Element, PromiseLike<WebElement> {
 
   getSize(): ElementValue<ScopedElementRect>;
 
+  rect(): ElementValue<ScopedElementRect>;
```

Comment:
It seems that this file does not contain the types for `getLocation`. Can you add that as well and group all 4 commands together?

Reply:
yep, was gonna ask about that, i also thought it was missing

Reply:
done 
bdd93d8